conduct/COC_TEAM_CHARTER.md

```diff
@@ -0,0 +1,92 @@
+This document defines how the OpenJS Foundation implements its code of conduct.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Team Charter
+
+The Cross Project Council (CPC) delegates enforcement of the [Code of Conduct][CoC] to the Code of Conduct Team (CoC Team) through this Code of Conduct Team Charter. This charter defines the composition and roles of the CoC Team, along with CoC team responsibilities and operations.
+
+## Responsibilities
+
+[TODO: add links]
+
+The CoC Team's responsibilities are to:
+
+- Handle code of conduct [incident reports][incident report].
+- Handle code of conduct [escalation requests][escalation].
+- Handle code of conduct [appeals][appeal].
+- Keep the CPC up to date of cases requiring the CPC's attention.
+- Provide regular transparency reports to the CPC.
+- Define, document, and maintain all processes necessary to its operations, notably the [CoC Process for Incident Management](COC_PROCESS_FOR_INCIDENT_MANAGEMENT.md).
```

Comment:
IMO self-definition is not the correct choice here, and I'm concerned that it's on the table. Any form of moderation team _should_ be people enabled to act on well-documented and well-defined processes and rules. To remove the potential for bias, **they should _not_ be charged with writing those rules themselves.**

In our case, writing these rules, documenting these rules, and maintaining the processes around those rules should be the charge of the CPC, not the CoC Team.

Reply:
That's the role of policy. The standard practice of good governance is to separate policy from process and offer a system to appeal the process if there's ground to believe it isn't correctly implementing policy. Moving process implementation to the CPC is the equivalent of asking legislators to author technical standards or shifting security right instead of left.

Reply:
Commented again on this here https://github.com/openjs-foundation/cross-project-council/pull/1135#pullrequestreview-1839768804 but I strongly and fundamentally disagree with this. I understand the concept you're operating off of but fundamentally I disagree with it and think that for community safety issues, it's an approach that opens up the opportunity for issues from both ends - it creates potential for abuse, and it creates the opportunity for decisions to be questioned.

Reply:
Repeating this comment here on suggestion from @tobie -

From the content proposed in this branch, there is still some work to do on [this particular file](https://github.com/openjs-foundation/cross-project-council/blob/coc/conduct/COC_TEAM_CHARTER.md) but the groundwork is there for eligibility. For this item, I think we could come to closure more efficiently by landing this PR and opening a specific issue (and tracking it on the CPC calls).

It does appear that the CoC Team composition and operations is a specific area that requires additional work. I propose we open this as a separate and specific issue so as not to block the entire CoC policy. That issue can be highlighted on our regular CPC calls-

Reply:
This was fixed. It is now the CPC which is in charge of this. Tracked in #1256.